lib/Dist/Zilla/App/Command/release.pm

```diff
@@ -11,10 +11,31 @@ use Dist::Zilla::App -command;
 
   dzil release --trial
 
+  # long form, jobs takes an integer
+  dzil release --jobs 9
+
+  # short form, same as above
+  dzil release --jobs 9
```

Comment:
Should the short form be `dzil release -j 9`?

Reply:
Oops, yes

Reply:
Fixed accordingly